client_cli/README.md

```diff
@@ -133,6 +133,28 @@ To know how many units of a particular asset an account has, use `asset get` wit
 
 This query returns the quantity of `XOR#Soramitsu` asset for the `White Rabbit@Soramitsu` account.
 
+It's possible to filter based on account, asset, domain id by using filtering api provided by client cli.
+
+Generally it looks like this:
+
+```bash
+./iroha_clien_cli ENTITY list filter PREDICATE
```

Comment:
```suggestion
./iroha_client_cli ENTITY list filter PREDICATE
```
And thinking about the structure of the command, it makes more sense to me if it was an option called with a flag rather than a subcommand, something like `--with-filter`. That would better bring back the point that it's optional IMO.

Reply:
I want to follow structure used later (`./iroha_client_cli ENTITY list all`). 
So we have two options either list everything or apply filter.